collections/search/index.php

```diff
@@ -567,31 +569,15 @@
 		ul.outerWidth(this.element.outerWidth());
 	}
 	const collectionSource = <?php echo isset($collectionSource) ? json_encode($collectionSource) : 'null'; ?>;
+	const collIdsFromUrl = <?php echo isset($collIdsFromUrl) ? json_encode($collIdsFromUrl) : 'null'; ?>;
+	if (collIdsFromUrl.length > 0) {
```

Comment:
this would need to be `collIdsFromUrl && collIdsFromUrl.length` to prevent a null pointer access

Reply:
Fixed in [b3cf85a](https://github.com/BioKIC/Symbiota/pull/1400/commits/b3cf85ab5c8964dc8a488bbd918b4d08fd814c6e)